frame/membership/src/lib.rs

```diff
@@ -264,6 +285,17 @@ impl<T: Config<I>, I: Instance> Module<T, I> {
 			}
 		}
 	}
+
+	fn maybe_warn_max_members(members: &[T::AccountId]) {
+		if members.len() as u32 > T::MaxMembers::get() {
+			log::error!(
+				target: "runtime::membership",
+				"max weight exceeded [{} > {}].",
```

Comment:
maybe a more explanatory error message,
```suggestion
				"maximum number of members used for weight is exceeded, weights can be underestimated [{} > {}].",
```